pkg/Microsoft.ML.Onnx/Microsoft.ML.Onnx.nupkgproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk" DefaultTargets="Pack">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <PackageDescription>ML.NET component for Exporting ONNX Models</PackageDescription>
```

Comment:
Do we envision using this package/library for more than just exporting an ONNX model? For example, if I want to load/execute an ONNX model, would that be through a separate library?

(nit) does `Exporting` need to be capitalized?

Reply:
> Do we envision using this package/library for more than just exporting an ONNX model? For example, if I want to load/execute an ONNX model, would that be through a separate library?

I'm not certain. If I had to guess, this hypothetical execution would involve linking to some native library in some fashion (Lotus?). I don't see much opportunity for code reuse in that case of course, so that suggests a different library.

> does Exporting need to be capitalized?

No.

Reply:
Sounds good.  We can adjust the package description later, if we add other functionality.